ocaml/doc/wire-protocol.md

```diff
@@ -56,7 +56,7 @@ These types are mapped onto XML-RPC types in a straight-forward manner:
   `(k -> v) map` type is only valid when `k` is a `string`, `ref`, or
   `int`, and in each case the keys of the maps are stringified as
   above. For example, the `(string -> double) map` containing the mappings
-  _Mike &rarr; 2.3_ and _John &rarr; 1.2_ would be represented as:
+  _Mike -> 2.3_ and _John -> 1.2_ would be represented as:
```

Comment:
Given the difference per system we may be safer with `->` after all; or even `&#45;&gt;` presuming the latter looks ok for you (may be worth double-checking the rest of the escaped characters in markdown_backen.ml).